src/main/java/org/dependencytrack/notification/publisher/JiraPublisher.java

```diff
@@ -0,0 +1,169 @@
+            final long epochSecond = notification.getTimestamp().toEpochSecond(
+                    ZoneId.systemDefault().getRules()
+                            .getOffset(notification.getTimestamp())
+            );
+            context.put("timestampEpochSecond", epochSecond);
+            context.put("timestamp", notification.getTimestamp().toString());
+            context.put("notification", notification);
+            if (baseUrlProperty != null) {
+                context.put("baseUrl", UrlUtil.normalize(baseUrlProperty.getPropertyValue()));
+            } else {
+                context.put("baseUrl", "");
+            }
+            if (NotificationScope.PORTFOLIO.name().equals(notification.getScope())) {
+                if (notification.getSubject() instanceof NewVulnerabilityIdentified) {
+                    final NewVulnerabilityIdentified subject = (NewVulnerabilityIdentified) notification.getSubject();
+                    context.put("subject", subject);
+                    context.put("subjectJson", NotificationUtil.toJson(subject));
+                } else if (notification.getSubject() instanceof NewVulnerableDependency) {
+                    final NewVulnerableDependency subject = (NewVulnerableDependency) notification.getSubject();
+                    context.put("subject", subject);
+                    context.put("subjectJson", NotificationUtil.toJson(subject));
+                } else if (notification.getSubject() instanceof AnalysisDecisionChange) {
+                    final AnalysisDecisionChange subject = (AnalysisDecisionChange) notification.getSubject();
+                    context.put("subject", subject);
+                    context.put("subjectJson", NotificationUtil.toJson(subject));
+                } else if (notification.getSubject() instanceof BomConsumedOrProcessed) {
+                    final BomConsumedOrProcessed subject = (BomConsumedOrProcessed) notification.getSubject();
+                    context.put("subject", subject);
+                    context.put("subjectJson", NotificationUtil.toJson(subject));
+                }
+            }
+            // Jira addition:
+            context.put("jiraProjectKey", jiraProjectKey);
+            context.put("jiraTicketType", jiraTicketType); //TEST
+
+            try (Writer writer = new StringWriter()) {
+                template.evaluate(writer, context);
+                return writer.toString();
+            } catch (IOException e) {
+                Logger.getLogger(this.getClass()).error("An error was encountered evaluating template", e);
+                return null;
+            }
+        }
+    }
+
+    @Override
+    public void publish(final String publisherName, final PebbleTemplate template, final Notification notification, final JsonObject config) {
+        final Logger logger = Logger.getLogger(this.getClass());
+        logger.info("Preparing to publish JIRA notification");
```

Comment:
Please remove or change log level to `debug`.

Reply:
done in https://github.com/elastic-pangolin/dependency-track/commit/17d7af47bc9d4bb1b74ce40b1a5af5e770e9c0be